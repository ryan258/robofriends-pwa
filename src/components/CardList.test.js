import React from 'react'

import CardList from './CardList'
import { shallow } from 'enzyme'

it('should render CardList component', () => {
  const mockRobots = [
    {
      id: 1,
      name: 'Fritzy McGee',
      email: 'fritzy@mcgee.com'
    }
  ]

  expect(shallow(<CardList robots={mockRobots} />)).toMatchSnapshot()
})
