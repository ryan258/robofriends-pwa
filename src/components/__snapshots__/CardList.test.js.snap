// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render CardList component 1`] = `
<div>
  <Card
    email="fritzy@mcgee.com"
    id={1}
    key="0"
    name="Fritzy McGee"
  />
</div>
`;
